### PR TITLE
Add namespace to workload get overview section

### LINKED ISSUE #321: add namespace to `...workload get ...` output
### Description of problem

For apps plugin users working across multiple namespaces, there will be scenarios where they need to know the namespace where the workload they're currently running the `workload get` command against is located.

For example
A dev is troubleshooting an issue with workload `my-app` and is trying to figure out at which point in their development the issue was introduced. They've deployed different versions of the workload to multiple namespaces in order to compare the behavior of each version.

If the namespace were included in the output of workload get, the dev could know at-a-glace, which version of `my-app` they were looking at.
If the namespace weren't included, they'd need to run a separate command (`tanzu apps workload get my-app --export`, or `kubectl get workload my-app -oyaml`), then parse the output to find the namespace.
This is a sub-optimal UX.

### Proposed solution

Provide the namespace in the output of workload get.

### Example
**workload get**
   ```
   tanzu apps workload get petc
   📡 Overview
      name:   petc
      type:   web
      namespace: foo
   ...
   ```
### Describe alternatives you've considered
1. Require the user to parse output of an --export
2. Update the output of `workload get` to prepend an info message which tells the user which namespace the command is being run against, something like "getting workload "my-app" details from namespace "foo"

### PR DESCRIPTION
**Pull request by warango4**
Pull request

### What this PR does / why we need it
Add namespace to workload get output in `Overview` section

### Which issue(s) this PR fixes

Fixes #321 

### Describe testing done for PR

- Updated unit testing

### Additional information or special notes for your reviewer

Signed-off-by: Wendy Arango <warango@vmware.com>

### CODE REVIEW

**Comment by warango4** [comment]
image

**Comment by warango4** [comment]
image

**Review by odinnordico — CHANGES_REQUESTED** [review]
Please update docs

**Review by shashwathi — COMMENTED** [review]
looks good. Update docs as mentioned in other comment.

**Review by shaheerkootteeri — COMMENTED** [review]
LGTM

File: docs/commands-details/workload_get.md [hunk 1]
```diff
@@ -8,63 +8,63 @@ You can view workload details at anytime in the process. Some details, such as t
 
 There are multiple sections in workload get command output. Following data is displayed
 
-- Name of the workload and its status.
-- Display source information of workload.
-- If the workload was matched with a supply chain, the information of its name and the status is displayed.
+- An overview section, where workload name, type and namespace where it's located are displayed.
```
Comment (shashwathi) [inline docs/commands-details/workload_get.md:11]: `workload name, type and namespace are displayed.`

File: docs/commands-details/workload_get.md [hunk 2]
```diff
@@ -8,63 +8,63 @@ You can view workload details at anytime in the process. Some details, such as t
...
+- Display workload source information.
+- If the workload was matched with a supply chain, the information of its name and the resources related to it are also displayed.
 - Information and status of the individual steps that's defined in the supply chain for workload.
 - Any issue with the workload, the name and corresponding message.
 - Workload related resource information and status like services claims, related pods, knative services.
 
-At the very end of the command output, a hint to follow up commands is also displayed.
+At the very end of the command output, a hint to follow up commands is also shown.
```
Comment (shashwathi) [inline docs/commands-details/workload_get.md:18]: keep the word `displayed`

**Review by odinnordico — APPROVED** [review]

**Pull request merged**